httpclient/src/main/java/com/baeldung/ClientTlsVersionExamples

```diff
@@ -0,0 +1,71 @@
+package com.baeldung
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.ssl.SSLContexts;
+import org.apache.http.util.EntityUtils;
+
+import javax.net.ssl.SSLSocket;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+
+public class ClientTlsVersionExmaples{
+
+    public static CloseableHttpClient setViaSocketFactory(){
+        SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(
+                SSLContexts.createDefault(),
+                new String[] {"TLSv1.2", "TLSv1.3"},
+                null,
+                SSLConnectionSocketFactory.getDefaultHostnameVerifier());
```

Comment:
these should indent only two spaces from where the code in line 19 begins

Reply:
fixed throughout file